src/sugar3/activity/activity.py

```diff
@@ -1251,7 +1262,16 @@ def get_activity_root():
     if os.environ.get('SUGAR_ACTIVITY_ROOT'):
         return os.environ['SUGAR_ACTIVITY_ROOT']
     else:
-        raise RuntimeError('No SUGAR_ACTIVITY_ROOT set.')
```

Comment:
Maybe the launcher could set the activity root as well?

Also, imports should be at the top of the file?